docs/products/clickhouse/howto/configure-backup.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Configure Aiven for ClickHouse® backup scheduling settings
+sidebar_label: Configure backups
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import RelatedPages from "@site/src/components/RelatedPages";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Set the time when
+[backups](/docs/products/clickhouse/concepts/disaster-recovery#service-backup) are taken
+for your Aiven for ClickHouse® service.
+
+## Prerequisites
+
+- Depending on what tools to use for configuring backup settings:
+  - [Aiven Console](https://console.aiven.io)
+  - [Aiven API](/docs/tools/api)
+  - [Aiven CLI](/docs/tools/cli)
+- At least one Aiven for ClickHouse service backup to be configured
+
+:::note
```

Comment:
What do you think about moving this note to the Configure backup time section? Right now, the note about resetting settings comes before the steps to configure them, which feels slightly out of order.